app-emulation/spice-protocol/spice_protocol-0.14.4.recipe

```diff
@@ -0,0 +1,42 @@
+SUMMARY="SPICE protocol definitions"
+DESCRIPTION="Headers defining the SPICE-protocol"
+HOMEPAGE="https://www.spice-space.org/"
+COPYRIGHT="2009 Red Hat, Inc"
+LICENSE="BSD (3-clause)"
+REVISION="1"
+SOURCE_URI="https://www.spice-space.org/download/releases/spice-protocol-$portVersion.tar.xz"
+CHECKSUM_SHA256="04ffba610d9fd441cfc47dfaa135d70096e60b1046d2119d8db2f8ea0d17d912"
+SOURCE_DIR="spice-protocol-$portVersion"
+
+ARCHITECTURES="any"
+
+PROVIDES="
+	spice_protocol = $portVersion
+	devel:spice_protocol = $portVersion
+"
+REQUIRES="
+	haiku
```

Comment:
If this package depends on this CR https://review.haiku-os.org/c/haiku/+/5745 then you should clearly state the pm version before this recipe gets merged.

Reply:
I'm not sure if buildmaster will build it in this case as it only has beta3 haiku package.

Reply:
hmm, for now, I simply fix it with patch **#ifdef SUN_LEN**